### DISCUSSION TITLE
闪退的看过来，临时解决办法

### DISCUSSION THREAD

**Discussion by lexjhuang** [getlantern/lantern-forum-cn#333] [General]
前天我的付费版突然闪退，经过一段调试后我写了一个bat脚本内容如下，之后我的付费版就可以正常启动了，希望这个对你们团队以及其他人有帮助！

del C:\Users\Administrator\AppData\Roaming\Lantern\global.yaml
Start C:\Users\Administrator\AppData\Roaming\Lantern\lantern.exe

根据我的经验，相信是global.yaml这个文件有编码错误！

**Comment by MaxMKMax** [comment]
@lexjhuang 这位大哥谢谢您! 感激不尽!
